src/updater.js

```diff
@@ -1,9 +1,16 @@
-import updateAttribute from './updateAttribute';
-import updateNode from './updateNode';
+import updateAttribute from "./updateAttribute";
+import updateNode from "./updateNode";
 
-import { applyHandlers } from './mountHandlerQueue';
+import { applyHandlers } from "./mountHandlerQueue";
 
-export default function updater (parts, values, oldValues = [], context, root) {
+export default function updater(
+  parts,
+  values,
+  oldValues = [],
+  context,
+  root,
+  forceUpdate
```

Comment:
Keep forceUpdate before root argument. You will also have to change all the places where updater is called.